app/service/family.py

```diff
@@ -145,3 +145,19 @@ def delete(import_id: str, db: Session = db_session.get_db()) -> bool:
         msg = f"Unable to delete family {import_id}"
         _LOGGER.exception(msg)
         raise RepositoryError(msg)
+    return family_repo.delete(db, import_id)
+
+
+@validate_call(config=ConfigDict(arbitrary_types_allowed=True))
```

Comment:
Looks like the analytics service uses the repo directly, so is there a need/use of this function?

Reply:
Updated analytics service to use the services instead of repositories.